docs/user_manual/boxes.md

```diff
@@ -0,0 +1,328 @@
+will refresh and any items with matching barcodes will be placed in their appropriate positions. Note that any
+items that were in the box before the Fill By Barcode Pattern was applied will be removed from the box.
+
+{% include userman-toplink.md %}
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Working with Items in a Box" %}
+
+Actions can be performed on all selected items in a box.
+
+First, go to the Edit Box page by clicking on the box name in the Boxes list page. 
+[Select the target positions](#selecting-positions-within-a-box) of the items you wish to act upon.
+In the toolbar at the top of the contents table, several actions are available for the selected items,
+provided that they are all of the same type (sample, library, dilution, pool). Note that not all actions are
+available for all item types.
+
+  * _Edit_: bulk edit the items
+  * _Propagate_: bulk propagate samples or libraries (only avialable for samples)
+  * _Make dilutions_: bulk propagate dilutions (only available for libraries)
+	* _Pool together_, _Pool separately_, _Pool custom_: [methods for creating pools](pools.html#pools_list)
+	  (only available for dilutions)
+  * _Create Orders_: bulk create orders for selected pools (only available for pools). See the [Orders
+    section](orders.html) for more information.
+  * _Print Barcode(s)_: print barcodes to a barcode printer. See the 
+    [Printing Barcodes section](barcode_label_printers.html#printing_barcodes) for more information.
+  * _Download_: download one or more spreadsheets for this type of item, if available
+  * _Parents_, _Children_: find items related to the selected items. See 
+    [Finding Related Items](general_navigation.html#finding_related_items) for more information.
+  * _Add QCs_, _Edit QCs_: quanlity control measures can be added to or edited for selected items. See
+    the [QCs section](qcs.html) for more information.
+  * _Add to Workset_: add these items to a new or existing workset. See the [Worksets section](worksets.html) 
+    for more information.
+  * _Attach File_: attach a single file to all of the selected items (not available for dilutions). See the
+    [Attachments section](attachments.html) for more information.
+
+
+{% include userman-toplink.md %}
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Printing Barcodes" %}
+
+Barcode labels can be printed for a series of boxes from the Boxes list. See the
+[Barcode Label Printers section - Printing Barcodes](barcode_label_printers.html#printing_barcodes) for
+details on how to do this.
+
+{% include userman-toplink.md %}
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Deleting Boxes" %}
+
+To delete boxes, go to the Boxes list, check the checkbox(es) next to the box(es) that you wish to delete, and
```

Comment:
Note that boxes may only be deleted by MISO administrators

Reply:
Anyone can delete a box.